### ISSUE TITLE
Classroom labs terraform

### ISSUE THREAD

**Issue by Stijnc** [Azure/azure-devtestlab#656]
Hi, 

I was wondering if terraform support is on the roadmap for classroom labs.
Currently there is only a PowerShell module (in preview) available if I understand correctly?
As an alternative are ARM templates available?

Thanks,
Stijn

**Comment by jkwon2013** [comment]
hi @Stijnc unfortunately there are no plans yet for terraform support yet. We will be working on improving and GA-ing our APIs though, so we are planning to do much improvement in the APIs (SDKs, PowerShell, CLI) space in the coming months.

**Comment by Stijnc** [comment]
Interesting and glad to hear.
I am now resorting to the python SDK to automate the creation of labaccounts and labs for on demand classrooms.

**Comment by jkwon2013** [comment]
Yes, and we know APIs are important for exactly those scenarios as customers scale :) thanks for providing the feedback

**Comment by Stijnc** [comment]
Hi @jkwon2013,
can you share any insights on the improvements you are planning?
